client/src/app/site/pages/meetings/pages/motions/services/export/motion-pdf.service/motion-pdf.service.ts

```diff
@@ -763,8 +764,8 @@ export class MotionPdfService {
                 attachments.push({
                     ul: [
                         {
-                            text: attachment.getTitle() + `: ` + instancUrl + fileUrl,
-                            link: instancUrl + fileUrl,
+                            text: attachment.getTitle() + `: ` + Location.joinWithSlash(instancUrl, fileUrl),
+                            link: Location.joinWithSlash(instancUrl, fileUrl),
```

Comment:
```suggestion
                const link = Location.joinWithSlash(instanceUrl, fileUrl);
                attachments.push({
                    ul: [
                        {
                            text: attachment.getTitle() + `: ` + link,
                            link: link,
```
While you're at it, please fix the variable name typo (`instancUrl` -> `instanceUrl`)